linkis-engineconn-plugins/hive/pom.xml

```diff
@@ -50,6 +50,12 @@
       <artifactId>linkis-storage</artifactId>
       <version>${project.version}</version>
       <scope>provided</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.hadoop</groupId>
+          <artifactId>hadoop-client</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
this just useful for compile.
i mean,  when start the ec,  the ec will include the classpath :  lib/linkis-commons/public-module,   which also have hadoop client.

Reply:
yes,indeed,if we need compile with hadoop3 ,the common modules will contains hadoop-clients3 for sure. maybe we need to make hive module and spark module fit for hadoop3, instand of keep old hdfs-client version